frontend/discourse/tests/unit/lib/lightbox/quote-image-test.js

```diff
@@ -27,6 +29,15 @@ module("Unit | Lib | lightbox | quote image", function (hooks) {
   setupTest(hooks);
 
   hooks.beforeEach(function () {
+    this.originalHelperContext = helperContext();
```

Comment:
Won't need all this if we check for current user in lightbox.js instead, then we can just add to the system spec for lightbox (if anon do not show quote button + do not show downlload button)